terraform/router/local/main.tf

```diff
@@ -1,33 +1,11 @@
-locals {
-  tags = {
-    System = "convox"
-    Rack   = var.name
-  }
-}
-
-module "nginx" {
```

Comment:
Why we need to remove this module?

Reply:
minikube dns feature requires ingress and ingress-dns to install. so ingress is already installed and we are not using this nginx ingress anymore that's why removed it